src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int LENGTH_OF_BY = 4;
+    private static final int LENGTH_OF_AT = 4;
+    private static final int LENGTH_OF_EVENT = 6;
+    private static final int LENGTH_OF_DEADLINE = 9;
+    private static final int LENGTH_OF_TODO = 5;
+
     public static void main(String[] args) {
+        String raw_input;
+        Scanner scanner = new Scanner(System.in);
+        TasksList tasks = new TasksList();
+
+        printWelcomeBanner();
+
+        raw_input = scanner.nextLine();
+        while (true) {
+            if (raw_input.equals("bye")) {  // Exit programme
+                System.out.println("\tBye. Hope to see you again soon! \uD83D\uDC4B");  // Unicode is a waving hand
+                printHorizontalLine();
+                break;
+            } else if (raw_input.startsWith("done ")) {  // Mark task as done
+                System.out.println("\tNice! \uD83D\uDC4D I've marked this task as done:");  // Unicode is a thumbs up
+                int taskIndex = Integer.parseInt(raw_input.substring(5))-1;
```

Comment:
You might like to insert spaces before and after arithmetic operator for clarity. I noticed the same issue in other lines in this class as well as TaskList.java.